approval_frame/forms.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import unicode_literals
+
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+from registration.forms import RegistrationFormUniqueEmail
+
+class RegistrationFormCustom(RegistrationFormUniqueEmail):
+    """
+    Subclass of `RegistrationFormUniqueEmail` which includes application-
+    specific customizations.
+
+    """
+    # Overriden to disallow '@' in the username during registration.
```

Comment:
This is an example of a good comment, as it is not easy to figure out by reading the code alone that usernames with the @ symbol are now invalid.